RoadMap_VK/RoadMap_VK/Cells/FriendsTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+// FriendsTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// Ячейка таблицы друзей
+final class FriendsTableViewCell: UITableViewCell {
+    // MARK: - Private visual elements
+
+    @IBOutlet private var friendPhotoImageView: UIImageView!
+    @IBOutlet private var friendView: FriendView!
+    @IBOutlet private var friendNameLabel: UILabel!
+
+    // MARK: - Public properties
+
+    var friendPhotoImageName: String?
```

Comment:
```suggestion
    private var friendPhotoImageName: String?
```
сделаем здесь приватное свойство для разнообразия